setup.py

```diff
@@ -41,6 +37,7 @@
         'click>=7.0',
         'h5py>=2.7.1',
         'matplotlib>=3.2.1',
+        'morphio>=2.3.10',
```

Comment:
I thought the plan was to move to MorphIO once it's ready - if it can be a dependency, does that mean it's ready?